### DISCUSSION TITLE
exception/try-with-resouces

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#141] [Announcements]
# exception/try-with-resouces

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，深入理解 Java 中的 try-with-resouces 语法糖

https://tobebetterjavaer.com/exception/try-with-resouces.html

**Comment by qinple** [comment]
之前只是见过在try后面加括号的"奇怪"写法, 原来是这个作用吗, 涨知识了.
